apps/app/src/utilities/global.ts

```diff
@@ -0,0 +1 @@
+export const leftPad = (n: number): string => n.toString().padStart(2, "0")
```

Comment:
This `leftPad` seems very specific for a single use-case. I think you could make it more generic by giving the ability of changing the parameters of `padStart` (`2, "0"`), instead of them being "hard-coded" here.

Reply:
[making leftpad more generic](https://github.com/gabrielakuhn/questline-galaxy/commit/fed73263e69a17f46d72b97e2205c8c0625b00bf)